packages/ui-app/src/CardGrid.tsx

```diff
@@ -13,14 +13,16 @@ type Props = I18nProps & {
   buttons?: React.ReactNode,
   children: React.ReactNode,
   className?: string,
-  emptyText?: string
+  emptyText?: string,
+  isEmpty?: boolean
 };
 
 class CardGrid extends React.PureComponent<Props> {
   render () {
-    const { buttons, children, className } = this.props;
+    const { buttons, children, className, isEmpty = false } = this.props;
 
-    if (!children || (children as Array<any>).length <= 0) {
+    console.log('children',children);
```

Comment:
For staking<actions the children is never empty as long as we have account, and the filtering whether something is shown or not is done after. 

The only way I thought of to make this work would be to filter the shown accounts upstream, this means bloating a little `/app-staking/src/index.tsx` with `derive.staking`. Any better idea?

Reply:
Can filter and always show, ie. If none pass empty array or null? Have not looked in detail.

Reply:
yes that's what I meant with "filtering upstream". I'll do that then.